pyproject.toml

```diff
@@ -0,0 +1,45 @@
+# With this the user can use a custom
+# python environment, like those created
+# with venv or pyenv. The deault will be
+# the system python.
+[virtualenvs]
+prefer-active-python = true
+
+# This isnt really needed on pratical aspect
+# since this is only for organized the functional tests
+# but is necessary since its a requirement for poetry
+[tool.poetry]
+name = "floresta-functional-tests"
+version = "0.0.1"
+description = "collection of tools to help with functional testing of Floresta"
+authors = [
```

Comment:
I think, for legal reasons, we should stick to the copyright notice in `LICENSE` and use `The Floresta Project Developers`

Reply:
fixed